Assets/Json/AbilityList.json

```diff
@@ -2,18 +2,170 @@
+    "Description": "Melee hits 2 characters.",
+    "Cooldown": 5.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 5,
+    "Name": "Cleave Ranged",
+    "ImageFilePath": "cleave ranged",
+    "Description": "Hits 2 characters from afar.",
+    "Cooldown": 5.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 6,
+    "Name": "Poison/DoT",
+    "ImageFilePath": "poisondot",
+    "Description": "Periodically does damage.",
+    "Cooldown": 6.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 7,
+    "Name": "Heal",
+    "ImageFilePath": "heal",
+    "Description": "Periodically increases an ally's health. Channelled over 4 seconds.",
+    "Cooldown": 4.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 8,
+    "Name": "Shield",
+    "ImageFilePath": "shield",
+    "Description": "Shields a single ally from damage.",
+    "Cooldown": 5.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 9,
+    "Name": "Sacrifice/Guard",
+    "ImageFilePath": "sacrificeguard",
+    "Description": "Shares 50% of all damage taken by an ally.",
+    "Cooldown": 8.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 10,
+    "Name": "Buff",
+    "ImageFilePath": "buff",
+    "Description": "Increases an ally's attribute.",
+    "Cooldown": 8.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 11,
+    "Name": "Debuff",
+    "ImageFilePath": "debuff",
+    "Description": "Weakens an opponent during the duration.",
+    "Cooldown": 6.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 12,
+    "Name": "Interrupt",
+    "ImageFilePath": "interrupt",
+    "Description": "Interrupts an opponents channelled ability.",
+    "Cooldown": 10.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 13,
+    "Name": "Silence",
+    "ImageFilePath": "silence",
+    "Description": "Prevents target from casting spells for 3 seconds. Can also interrupt.",
+    "Cooldown": 12.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 14,
+    "Name": "Confuse",
+    "ImageFilePath": "confuse",
+    "Description": "Target can't identify friend from foe for 3 seconds.",
+    "Cooldown": 15.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 15,
+    "Name": "Demoralize",
+    "ImageFilePath": "demoralize",
+    "Description": "Target receives 30% more damage over the next 3 seconds.",
+    "Cooldown": 12.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 16,
+    "Name": "Rampage",
+    "ImageFilePath": "rampage",
+    "Description": "An ally receives a 40% attack speed increase for the next 4 seconds.",
+    "Cooldown": 12.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 17,
+    "Name": "Flurry",
+    "ImageFilePath": "flurry",
+    "Description": "Character does 4 attacks in quick succession.",
```

Comment:
Væri ágætt að taka fram hvernig attack, weapon/basic etc.
Veit að það er kannski óákveðið hjá okkur núna, en gott að hafa í huga.

Reply:
Gera það Basic Attack bara? (for now)